packages/flutter/lib/src/widgets/title.dart

```diff
@@ -13,12 +13,17 @@ class Title extends StatelessWidget {
   /// Creates a widget that describes this app to the operating system.
   Title({
     Key key,
-    this.title,
+    this.title: kDefaultTitle,
     this.color,
     @required this.child,
   }) : assert(color == null || color.alpha == 0xFF),
+       assert(title != null),
        super(key: key);
 
+  /// The default title used by this widget, and by the App widgets
+  /// ([MaterialApp], [WidgetsApp], etc.) that use this one.
+  static const String kDefaultTitle = '';
```

Comment:
you can just hardcode `''` everywhere. We don't usually both with this kind of indirection unless the value is non-trivial (i.e. not null, 0, false, `''`, or similar).